webviewer/src/pages/Account/components/Sidebar/Sidebar.jsx

```diff
@@ -3,19 +3,17 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { ROUTES } from '_constants';
 import { TABS } from '../../constants';
-import { StyledSidebar, BreadCrumbsItem, BreadCrumbs, TabItem } from './Sidebar.styles';
+import { StyledSidebar, StyledBreadcrumbs, TabItem } from './Sidebar.styles';
 
 export const Sidebar = props => {
   const { activeTab, setActiveTab } = props;
   const history = useHistory();
-  const handleClickHome = () => history.push(ROUTES.ROOT);
 
   return (
     <StyledSidebar>
-      <BreadCrumbs>
-        <BreadCrumbsItem onClick={handleClickHome}>Home</BreadCrumbsItem> /{' '}
-        <BreadCrumbsItem current>Personal account</BreadCrumbsItem>
-      </BreadCrumbs>
+      <StyledBreadcrumbs
+        items={[{ link: ROUTES.ROOT, text: 'Home' }, { text: 'Personal account' }]}
```

Comment:
maybe defined items as constant and extract it from component render?

Reply:
done